client/src/modules/Registry/components/Form/GeneralSection/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { Dispatch, SetStateAction } from 'react';
+import css from 'styled-jsx/css';
+import { Location } from 'common/models';
+import { PersonalInfo } from '../../PersonalInfo';
+import { ContactInfo } from '../../ContactInfo';
+
+type Props = {
+  location: Location | null;
+  setLocation: Dispatch<SetStateAction<Location | null>>;
+};
+
+const styles = css`
```

Comment:
Please remove custom styles and use antdesign:
* https://ant.design/components/form#components-form-demo-layout

Reply:
fixed